src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/RestrictedPatientsService.kt

```diff
@@ -73,6 +75,49 @@ class RestrictedPatientsService(
       .let { addRestrictedPatient(it) }
   }
 
+  @Transactional
+  fun prisonerReleased(offenderNo: String) {
```

Comment:
The main change is to add a new service method that will add prisoners who were released to a psych hospital in NOMIS into Restricted Patients.

Note this is not wired in anywhere yet.